Paws4Me/Paws4Me/Show and Validate Login/View Models/SignInViewModel.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ViewModels.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/24.
+//
+
+import Foundation
+
+protocol Validation {
+    func isValidCredentials(username: String, password: String) -> Bool
+}
+
+class SignInViewModel: Validation {
+
+    // MARK: - Functions
+    func isValidCredentials(username: String, password: String) -> Bool {
+        if username == "Admin" && password == "TestPass123" {
```

Comment:
This can be simplified to: 

`return username == "Admin" && password == "TestPass123"`

Reply:
That is much better. Thank you for the suggestion 👍🏻